### PR TITLE
Added intro text

### PR DESCRIPTION
**Pull request by AustinKobayashi**

### CODE REVIEW

File: plugins/firebase.js [hunk 1]
```diff
@@ -28,6 +28,28 @@ const fireDb = {
...
   },
+  getIntroText: async () => {
+    const websites = await fireDb.getWebsites()
+    const introTexts = {}
+    for (const website of websites) {
+      const websiteData = (await db
+        .collection(webCollection)
+        .doc(website)
+        .get()).data()
+      introTexts[website] = {
```
Comment (Ciuffi) [inline plugins/firebase.js:39]: Just wondering why this is a string array instead of just a string that reloads when you select another wbesite?

**Review by Ciuffi — COMMENTED** [review]
This looks pretty good but I can't test if it works until auth is implemented.

**Review by Ciuffi — COMMENTED** [review]
Add timestamp and last edited by

File: plugins/firebase.js (hunk 1 above)
Comment (AustinKobayashi) [reply plugins/firebase.js:39]: Just a design choice. We could just get all website intro texts, and create a dictionary of <website, intro-stuffs> when the cms page is loaded. Or when the user changes a website, we can get the intro text values, then set them on the cms page. First option only requires one firebase call so I went with that.

Comment (Ciuffi) [reply plugins/firebase.js:39]: Good call, thanks for explaining!

**Review by Ciuffi — APPROVED** [review]
Very nice thank you for adding those features!

**Pull request merged**